extractor/TextExtractor_onepage.py

```diff
@@ -0,0 +1,48 @@
+from abc import ABC, abstractmethod
+from typing import List
+from bs4 import BeautifulSoup
+import requests
+import textwrap
+
+class Extractor(ABC):
```

Comment:
Schön mit der abstract class!

Die `extract`-Methode braucht allerdings noch einen return-type, damit für alle Zugriffe der  output klar ist.

Return types definierst du wie in der Implementation im `WikipediaExtractor` mit dem Pfeil `->` nach der Klammer.